data/Mazeedkmk204.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Mazeed Khan",
+  "bio": "Open Source DevRel | Frontend Developer | **GitHub** Star",
+  "links": [
+    {
+      "group": "Business",
+      "name": "jabeen taj automobiles",
```

Comment:
You forgot to add your URL here... like this `"url": "http://examplelink.io",`

Reply:
link to the docs if you need them [Profile with JSON](https://www.biodrop.io/docs/quickstart-json)